get.sh

```diff
@@ -556,6 +556,9 @@ getFunctionalTestMaterial()
 				echo "Stage $dest/$dir to $TESTDIR/$dir"
 				# already in TESTDIR, thus copy $dir to current directory
 				cp -r $dest/$dir ./
+				if [[ "$PLATFORM" == "s390x_zos" ]]; then
```

Comment:
We also need to consider 31bit zos. Please check if $PLATFORM contains `zos`

Reply:
Modified the check to address 31/64 bit z/OS. 
Verification job 
hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/15741